src/plugins/intel_gpu/src/graph/layout_optimizer.cpp

```diff
@@ -1833,6 +1831,23 @@ format layout_optimizer::get_preferred_format(program_node& node) {
                 expected = format::b_fs_yx_fsv32;
             }
         }
+    } else if (node.is_type<reduce>()) {
```

Comment:
I think we need to check `use_onednn_impls`, too

Reply:
Applied